### PR TITLE
Fix/kriii/get user nft list

### PR DESCRIPTION
**Pull request by kriii**
Sorry, I just realized that I shouldn't change the code like [this](https://github.com/ainft-team/ainft-js/pull/67/commits/a001489e9bbc7a5131095b5fadec7c5141cd2138). So, I changed args and typing.

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -336,21 +336,30 @@ export type NftTokens = {
...
   chain: string,
   name: string,
   symbol: string,
   contractAddress: string,
+  deployAddress: string,
+  totalSupply: number,
+  lastBlockNumber: number,
+};
+
+export type NftContract = {
```
Comment (platfowner) [inline src/types.ts:349]: Is this change backward compatible?

Comment (kriii) [reply src/types.ts:349]: Should we need backward compatibility after upgrading the version? I thought this typing can notify that should need to change about this.

Comment (kriii) [reply src/types.ts:359]: I reverted backward compatibility!

**Comment by platfowner** [comment]
cc: @jakepyo @shyun-comcom @yoojinko @Laeyoung @minsulee2

**Review by platfowner — APPROVED** [review]
LGTM
Thanks for working on it!

**Comment by kriii** [comment]
Thanks! Now merged!

**Pull request merged**